cdp_scrapers/instances/portland.py

```diff
@@ -257,23 +361,29 @@ def get_event_minutes(
 
         # think we will let MinutesItem.name = Matter.name
         #                   MinutesItem.description = Matter.title
-        return reduced_list(
-            [
-                self.get_none_if_empty(
-                    EventMinutesItem(
-                        decision=None,
-                        index=0,
-                        matter=self.get_matter(event_page),
-                        minutes_item=MinutesItem(name=None, description=None),
-                        # TODO: need to call get_supporting_files()
-                        # with the minutes item's index on the current event page
-                        # or some similar minutes item differentiator
-                        # from other minutes items on the event page
-                        # supporting_files=self.get_supporting_files(event_page),
-                        votes=self.get_votes(event_page),
-                    ),
+
+        minute_sections = event_page.find_all(
+            "div", class_="relation--type-agenda-item"
+        )
+        event_minute_items = []
+        for minute_section in minute_sections:
+            title_div = minute_section.find("div", class_="council-document__title")
+            matter_title = title_div.find("a").text.strip()
+            index = int(minute_section.find("h4").text.strip())
+            event_minute_item = EventMinutesItem(
+                decision=None,
+                index=index,
+                matter=self.get_matter(minute_section),
+                minutes_item=self.get_none_if_empty(
+                    MinutesItem(name=None, description=matter_title)
                 ),
-            ],
+                # supporting_files=self.get_supporting_files(minute_section, index),
+                votes=self.get_votes(minute_section),
+            )
+            event_minute_items.append(self.get_none_if_empty(event_minute_item))
+
+        return reduced_list(
+            [event_minute_items],
```

Comment:
This is double-listing, Shak, right? Isn't `event_minute_items` already a `list`? i.e. This should just be `return reduced_list(event_minute_items)`?

Reply:
You're right! I removed the brackets.